src/Ocelot/Configuration/Builder/AuthenticationOptionsBuilder.cs

```diff
@@ -7,6 +7,8 @@ public class AuthenticationOptionsBuilder
         private List<string> _allowedScopes = new();
         private string _authenticationProviderKey;
 
+        private List<string> _authenticationProviderKeys = new();

```

Comment:
```suggestion
        private string[] _authenticationProviderKeys = new();

```

Reply:
Yeah, it seems we're going to use arrays in [documentation](https://ocelot.readthedocs.io/en/latest/features/authentication.html?highlight=AuthenticationProviderKey).